src/widgets/refinement-list/refinement-list.tsx

```diff
@@ -20,41 +21,42 @@ import { Template, WidgetFactory, RendererOptions } from '../../types';
 import { PreparedTemplateProps } from '../../lib/utils/prepareTemplateProps';
 import searchBoxDefaultTemplates from '../search-box/defaultTemplates';
 import { SearchBoxTemplates } from '../search-box/search-box';
+import { SearchBoxComponentTemplates } from '../../components/SearchBox/SearchBox';
 
 const withUsage = createDocumentationMessageGenerator({
   name: 'refinement-list',
 });
 const suit = component('RefinementList');
 const searchBoxSuit = component('SearchBox');
 
-type RefinementListOwnTemplates = {
+export type RefinementListOwnTemplates = {
```

Comment:
If we export it, we can rename it to that for consistency IMO:

```suggestion
export type RefinementListTemplates = {
```

Reply:
I've also had this concern with the `CSSClasses` types but it only refer to the `refinement-list` widget itself, without the `searchable` templates so that's why I've decided to keep this naming.

[`RefinementListTemplates`](https://github.com/algolia/instantsearch.js/blob/74c7158c98a6f0d058e113687d072f570fedc518/src/widgets/refinement-list/refinement-list.tsx#L62-L63) is the other existing type for both the `OwnTemplates` & the searchable ones

Reply:
We only want to export the complete type then, not this non-complete one?

Reply:
It's now exported to be used in the `Component` and create the `RefinementListComponentTemplates` type, do you think it shouldn't be exposed at all? (https://github.com/algolia/instantsearch.js/pull/4785/commits/e731f890f197e56576e017f47a721b60e30c54b5)

In that case, I could create the `Component` type directly from the widget 🤔